composer.json

```diff
@@ -28,5 +28,17 @@
         "psr-4": {
             "Facebook\\InstantArticles\\": "tests/Facebook/InstantArticles/"
         }
+    },
+    "scripts": {
+        "all": [
+            "composer cs",
+            "composer test"
+        ],
+        "cs": [
+            "vendor/bin/phpcbf --standard=phpcs.xml -p || vendor/bin/phpcs --standard=phpcs.xml -p"
+        ],
+        "test": [
```

Comment:
We could "depend" on install by adding `"@composer install",` as in the example [here](https://getcomposer.org/doc/articles/scripts.md#calling-composer-commands)